File: E28D_20190813.pas
//  
//  DB Script Tool
//  Delphi - 2019-08-13 11:46:57
//  
//  MODEL CLASSES FOR E28D DATABASE
//



(* TSYO.pas -------------------------------------*)
unit TSYOUnit;

interface

uses
    Classes, System.DateUtils;

/// <summary>
///  Delphi - Model Class - TSYO
///  SYO
/// </summary>
type
    TSYO = class(TObject)

    private
    SYO : LongInt;

    public
        //default constructor
        //constructor Create; overload; override;
        constructor Create(m_SYO: LongInt);
    property SYO : LongInt read GetSYO write SetSYO;
        function ToString : String;
    end;

//const
//    CONST_SAMPLE = 1;  // Integer constant

//var
//    var_sample: Integer;

implementation

uses
    System.SysUtils, System.DateUtils, System.StrUtils;

// -----------------------
// Constructor
// Example: 
// mySYO : TSYO;
// mySYO := TSYO.Create;
// -----------------------
//constructor TSYO.Create;
//begin
    //--
//end;


// -----------------------
// Constructor
// Example: 
// mySYO : TSYO;
// mySYO := TSYO.Create( val1, val2,.. );
// -----------------------
constructor TSYO.Create(m_SYO: LongInt);
begin
        self.SYO := m_SYO;
end;


// --
// Getters and Setters
// --

function TSYO.GetSYO : LongInt;
begin
    Result := self.SYO;
end;

procedure TSYO.SetSYO(m_SYO : LongInt);
begin
    self.SYO := m_SYO;
end;



// --
// Methods
// --

// Get a description string from this object
function SYO.ToString : String;
begin
    Result := '';
end;

end. // end of the unit



(* TS.pas -------------------------------------*)
unit TSUnit;

interface

uses
    Classes, System.DateUtils;

/// <summary>
///  Delphi - Model Class - TS
///  S
/// </summary>
type
    TS = class(TObject)

    private
    S : LongInt;

    public
        //default constructor
        //constructor Create; overload; override;
        constructor Create(m_S: LongInt);
    property S : LongInt read GetS write SetS;
        function ToString : String;
    end;

//const
//    CONST_SAMPLE = 1;  // Integer constant

//var
//    var_sample: Integer;

implementation

uses
    System.SysUtils, System.DateUtils, System.StrUtils;

// -----------------------
// Constructor
// Example: 
// myS : TS;
// myS := TS.Create;
// -----------------------
//constructor TS.Create;
//begin
    //--
//end;


// -----------------------
// Constructor
// Example: 
// myS : TS;
// myS := TS.Create( val1, val2,.. );
// -----------------------
constructor TS.Create(m_S: LongInt);
begin
        self.S := m_S;
end;


// --
// Getters and Setters
// --

function TS.GetS : LongInt;
begin
    Result := self.S;
end;

procedure TS.SetS(m_S : LongInt);
begin
    self.S := m_S;
end;



// --
// Methods
// --

// Get a description string from this object
function S.ToString : String;
begin
    Result := '';
end;

end. // end of the unit



(* T1440.pas -------------------------------------*)
unit T1440Unit;

interface

uses
    Classes, System.DateUtils;

/// <summary>
///  Delphi - Model Class - T1440
///  1440
/// </summary>
type
    T1440 = class(TObject)

    private
    1440 : LongInt;

    public
        //default constructor
        //constructor Create; overload; override;
        constructor Create(m_1440: LongInt);
    property 1440 : LongInt read Get1440 write Set1440;
        function ToString : String;
    end;

//const
//    CONST_SAMPLE = 1;  // Integer constant

//var
//    var_sample: Integer;

implementation

uses
    System.SysUtils, System.DateUtils, System.StrUtils;

// -----------------------
// Constructor
// Example: 
// my1440 : T1440;
// my1440 := T1440.Create;
// -----------------------
//constructor T1440.Create;
//begin
    //--
//end;


// -----------------------
// Constructor
// Example: 
// my1440 : T1440;
// my1440 := T1440.Create( val1, val2,.. );
// -----------------------
constructor T1440.Create(m_1440: LongInt);
begin
        self.1440 := m_1440;
end;


// --
// Getters and Setters
// --

function T1440.Get1440 : LongInt;
begin
    Result := self.1440;
end;

procedure T1440.Set1440(m_1440 : LongInt);
begin
    self.1440 := m_1440;
end;



// --
// Methods
// --

// Get a description string from this object
function 1440.ToString : String;
begin
    Result := '';
end;

end. // end of the unit



(* TSY.pas -------------------------------------*)
unit TSYUnit;

interface

uses
    Classes, System.DateUtils;

/// <summary>
///  Delphi - Model Class - TSY
///  SY
/// </summary>
type
    TSY = class(TObject)

    private
    SY : LongInt;

    public
        //default constructor
        //constructor Create; overload; override;
        constructor Create(m_SY: LongInt);
    property SY : LongInt read GetSY write SetSY;
        function ToString : String;
    end;

//const
//    CONST_SAMPLE = 1;  // Integer constant

//var
//    var_sample: Integer;

implementation

uses
    System.SysUtils, System.DateUtils, System.StrUtils;

// -----------------------
// Constructor
// Example: 
// mySY : TSY;
// mySY := TSY.Create;
// -----------------------
//constructor TSY.Create;
//begin
    //--
//end;


// -----------------------
// Constructor
// Example: 
// mySY : TSY;
// mySY := TSY.Create( val1, val2,.. );
// -----------------------
constructor TSY.Create(m_SY: LongInt);
begin
        self.SY := m_SY;
end;


// --
// Getters and Setters
// --

function TSY.GetSY : LongInt;
begin
    Result := self.SY;
end;

procedure TSY.SetSY(m_SY : LongInt);
begin
    self.SY := m_SY;
end;



// --
// Methods
// --

// Get a description string from this object
function SY.ToString : String;
begin
    Result := '';
end;

end. // end of the unit



(* TYO.pas -------------------------------------*)
unit TYOUnit;

interface

uses
    Classes, System.DateUtils;

/// <summary>
///  Delphi - Model Class - TYO
///  YO
/// </summary>
type
    TYO = class(TObject)

    private
    YO : LongInt;

    public
        //default constructor
        //constructor Create; overload; override;
        constructor Create(m_YO: LongInt);
    property YO : LongInt read GetYO write SetYO;
        function ToString : String;
    end;

//const
//    CONST_SAMPLE = 1;  // Integer constant

//var
//    var_sample: Integer;

implementation

uses
    System.SysUtils, System.DateUtils, System.StrUtils;

// -----------------------
// Constructor
// Example: 
// myYO : TYO;
// myYO := TYO.Create;
// -----------------------
//constructor TYO.Create;
//begin
    //--
//end;


// -----------------------
// Constructor
// Example: 
// myYO : TYO;
// myYO := TYO.Create( val1, val2,.. );
// -----------------------
constructor TYO.Create(m_YO: LongInt);
begin
        self.YO := m_YO;
end;


// --
// Getters and Setters
// --

function TYO.GetYO : LongInt;
begin
    Result := self.YO;
end;

procedure TYO.SetYO(m_YO : LongInt);
begin
    self.YO := m_YO;
end;



// --
// Methods
// --

// Get a description string from this object
function YO.ToString : String;
begin
    Result := '';
end;

end. // end of the unit

File: E28D_20190813.cpp
/* 
*  DB Script Tool
*  C++ - 2019-08-13 11:47:10
*  
*  MODEL CLASSES FOR E28D DATABASE
*/ 



/* SYO h / cpp -------------------------------------*/
#include <iostream>        // using IO functions
#include <string>            // using string

using namespace std; 

/* C++ - Model Class Definition - E28D.SYO */

class SYO {

private: 
    int m_SYO;

public: 
    SYO();
    SYO(int SYO);

    int getSYO() const;
    void setSYO(int SYO);

    string toString();
};

/* C++ - Model Class Implementation - E28D.SYO */

/**
* Constructor
* 
* Example: 
* SYO mySYO = new SYO();
*/
SYO::SYO() {
    //--
}

/**
* Constructor
* 
* Example: 
* SYO mySYO = new SYO( val1, val2,.. );
*/
SYO::SYO(int SYO) {
    this->m_SYO = SYO;
}


/**
* Getters and Setters
*/

int SYO::getSYO() const {
    return this->m_SYO;
}

void SYO::setSYO(int SYO) {
    this->m_SYO = SYO;
}



/**
* Methods
*/

string SYO::toString() {
    return "";
}



/* S h / cpp -------------------------------------*/
#include <iostream>        // using IO functions
#include <string>            // using string

using namespace std; 

/* C++ - Model Class Definition - E28D.S */

class S {

private: 
    int m_S;

public: 
    S();
    S(int S);

    int getS() const;
    void setS(int S);

    string toString();
};

/* C++ - Model Class Implementation - E28D.S */

/**
* Constructor
* 
* Example: 
* S myS = new S();
*/
S::S() {
    //--
}

/**
* Constructor
* 
* Example: 
* S myS = new S( val1, val2,.. );
*/
S::S(int S) {
    this->m_S = S;
}


/**
* Getters and Setters
*/

int S::getS() const {
    return this->m_S;
}

void S::setS(int S) {
    this->m_S = S;
}



/**
* Methods
*/

string S::toString() {
    return "";
}



/* 1440 h / cpp -------------------------------------*/
#include <iostream>        // using IO functions
#include <string>            // using string

using namespace std; 

/* C++ - Model Class Definition - E28D.1440 */

class 1440 {

private: 
    int m_1440;

public: 
    1440();
    1440(int 1440);

    int get1440() const;
    void set1440(int 1440);

    string toString();
};

/* C++ - Model Class Implementation - E28D.1440 */

/**
* Constructor
* 
* Example: 
* 1440 my1440 = new 1440();
*/
1440::1440() {
    //--
}

/**
* Constructor
* 
* Example: 
* 1440 my1440 = new 1440( val1, val2,.. );
*/
1440::1440(int 1440) {
    this->m_1440 = 1440;
}


/**
* Getters and Setters
*/

int 1440::get1440() const {
    return this->m_1440;
}

void 1440::set1440(int 1440) {
    this->m_1440 = 1440;
}



/**
* Methods
*/

string 1440::toString() {
    return "";
}



/* SY h / cpp -------------------------------------*/
#include <iostream>        // using IO functions
#include <string>            // using string

using namespace std; 

/* C++ - Model Class Definition - E28D.SY */

class SY {

private: 
    int m_SY;

public: 
    SY();
    SY(int SY);

    int getSY() const;
    void setSY(int SY);

    string toString();
};

/* C++ - Model Class Implementation - E28D.SY */

/**
* Constructor
* 
* Example: 
* SY mySY = new SY();
*/
SY::SY() {
    //--
}

/**
* Constructor
* 
* Example: 
* SY mySY = new SY( val1, val2,.. );
*/
SY::SY(int SY) {
    this->m_SY = SY;
}


/**
* Getters and Setters
*/

int SY::getSY() const {
    return this->m_SY;
}

void SY::setSY(int SY) {
    this->m_SY = SY;
}



/**
* Methods
*/

string SY::toString() {
    return "";
}



/* YO h / cpp -------------------------------------*/
#include <iostream>        // using IO functions
#include <string>            // using string

using namespace std; 

/* C++ - Model Class Definition - E28D.YO */

class YO {

private: 
    int m_YO;

public: 
    YO();
    YO(int YO);

    int getYO() const;
    void setYO(int YO);

    string toString();
};

/* C++ - Model Class Implementation - E28D.YO */

/**
* Constructor
* 
* Example: 
* YO myYO = new YO();
*/
YO::YO() {
    //--
}

/**
* Constructor
* 
* Example: 
* YO myYO = new YO( val1, val2,.. );
*/
YO::YO(int YO) {
    this->m_YO = YO;
}


/**
* Getters and Setters
*/

int YO::getYO() const {
    return this->m_YO;
}

void YO::setYO(int YO) {
    this->m_YO = YO;
}



/**
* Methods
*/

string YO::toString() {
    return "";
}